### ISSUE TITLE
Rewrite class attributes on components

### ISSUE THREAD

**Issue by jacojoubert** [prototypal-io/untitled-ui#51]
Instead of doing:

``` hbs
{{ui-component class="ui-select--default"}}
```

we should be able to do

``` hbs
{{ui-component class=":component"}}
```

**Comment by ebryn** [comment]
Hmm, can you explain the use case?

**Comment by jacojoubert** [comment]
There is an inconsistency when you use classes with 3rd party components. Take a look at https://github.com/prototypal-io/untitled-ui/blob/e69115c617e32a70b350d9a622026e73ccab6ecd/addon/templates/components/ui-select--default.hbs#L2

**Comment by ianstarz** [comment]
Blocked by rewriters improvements; will have to look at recursing into mustache statements.

**Comment by ebryn** [comment]
Closing in favor of #53, let's make sure this is covered by that work.
